pipelines/release-pipeline.yml

```diff
@@ -261,7 +261,7 @@ stages:
               {
                 "KeyCode" : "CP-401337-Apple",
                 "OperationCode" : "MacAppNotarize",
-                "Parameters" : {"BundleId":"com.microsoft.Azure.IoTExplorer"},
+                "Parameters" : {"BundleId":"com.microsoft.azure.iot.pnp.ui"},
```

Comment:
I am not quite sure about this value change. Is it something we changed on the pipeline side?